fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationOptionsDialogBottomSheet.kt

```diff
@@ -52,53 +47,45 @@ fun TranslationOptionsDialogBottomSheet(
     onTranslationSettingsClicked: () -> Unit,
     aboutTranslationClicked: () -> Unit,
 ) {
-    Column(
```

Comment:
I think we could do some renaming/refactoring here. The "bottom sheet" in my mind is the top-level `Column` with rounded corners at the top and maybe the header with the back button. I think what are showing here now is just the contents (eg the list of translation options) of the bottom sheet and doesn't necessary contribute to the look of a bottom sheet. 

What I would like to see is:

1. The `LazyColumn` containing the `translationOptionsList` should just be a Composable named `TranslationOptions` or some variant of this name.
2. Looking at the `Column` that is defined in the Fragment at [L93](https://github.com/mozilla-mobile/firefox-android/pull/4286/files#diff-9554cfcf3e9c0249875f28de8699b2cca3beddce81be44d4c4a2cf706772449cR93), I am wondering if we refactor that into a Composable function that is named `TranslationDialogBottomSheet`?

Reply:
Done

Reply:
Let's rename this file and the Composable to be `TranslationOptions` or `TranslationOptionsDialog` since we have a top-level `TranslationDialogBottomSheet` now, and the components here only represent the content of the bottom sheet and isn't the entire bottom sheet + content in itself based on how it is currently named.